sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -336,18 +336,18 @@ def __init__(self, proto_message_type):
     self.proto_message_type = proto_message_type
 
   def encode(self, value):
-    return value.SerializeToString()
+    return value.SerializePartialToString()
 
   def decode(self, encoded):
     proto_message = self.proto_message_type()
-    proto_message.ParseFromString(encoded)
+    proto_message.ParseFromString(encoded)  # This is in effect "ParsePartial".
```

Comment:
`# This is in effect "ParsePartial".` -> Why is this the case?

Reply:
This comment is saying that this method can parse both initialized and partial messages. (there is no ParsePartialFromString)